### ISSUE TITLE
[Request] Add npm installation instructions to readme.md

### ISSUE THREAD

**Issue by Tibfib** [TheSharpieOne/angular-off-click#43]
This is a great directive. Right now I've downloaded the source file for inclusion in my angular app, but I'd love to just have it via npm. I don't use bower (and neither do many frontend devs these days) so that doesn't work for me.

Thanks for this!

**Comment by Tibfib** [comment]
Hah shoot. It _is_ on npm... but hard to find. Could you indicate that on the readme.md?

**Comment by TheSharpieOne** [comment]
Someone else had put this on npm and I am not a collaborator for the package on npm... The npm package is not even the latest. If I were given control to the package I would do that. I don't believe creating another package on npm with a different name will help the matter as it would only become more confusing.

**Comment by Ricardo-Marques** [comment]
@TheSharpieOne You should publish it under a namespace http://blog.npmjs.org/post/116936804365/solving-npms-hard-problem-naming-packages

**Comment by Tibfib** [comment]
I am in agreement there. This would be great to have on npm.

**Comment by TheSharpieOne** [comment]
I contacted the person who published the package on npm and they granted me access. I have published the latest version to npm as well as updated the README.md to include the npm installation instructions . I have also added badges which indicate the current versions on each of the package repositories with links to them.

@Ricardo-Marques what is your npm username so I can grant you access so you will be able to publish new version when needed.
